src/data-source.js

```diff
@@ -6,6 +6,7 @@ import { UserSchema } from './models/user.js';
 import { RefreshTokenSchema } from './models/refreshToken.js';
 import { OperatorSchema } from './operatorApp/models.js';
 import { ResetTokenSchema } from './userApp/models.js';
+import { routeSchema } from './routeApp/models.js';
 
```

Comment:
For consistency, you can rename your `routeSchema` to match other schema but not a big a deal really